ci/jenkins/test-vmc.sh

```diff
@@ -353,6 +353,13 @@ function deliver_antrea {
     # Clean up dangling images generated in previous builds. Recent ones must be excluded
     # because they might be being used in other builds running simultaneously.
     docker image prune -f --filter "until=1h" > /dev/null
```

Comment:
do you know why we use `docker image prune` instead of `docker system prune`.
Maybe we could just use something like `docker system prune --force --all --filter until=4h`?

Reply:
@antoninbas Since all tests currently can be completed within one hour, I am considering using `docker image prune -af --filter "until=1h"` to remove all images not used within the past hour. What do you think?

Reply:
ok good to know

Reply:
> @antoninbas Since all tests currently can be completed within one hour, I am considering using `docker image prune -af --filter "until=1h"` to remove all images not used within the past hour. What do you think?

Hmm, could we do 2h for this one to have a buffer?
```
docker image prune -af --filter "until=2h"
docker image prune -f --filter "until=1h"
```

Reply:
> @antoninbas Since all tests currently can be completed within one hour, I am considering using `docker image prune -af --filter "until=1h"` to remove all images not used within the past hour. What do you think?

Sounds good to me. We do not allow concurrent jobs anyway, so this should always be safe?

Reply:
> Sounds good to me. We do not allow concurrent jobs anyway, so this should always be safe?

During the image building phase, we allow three jobs to run in parallel. But during the deployment and testing phases, each job has its own test cluster created by CAPV.

Reply:
Building the image on the capv control node typically takes around 10 minutes after the job starts. Once the image is copied to the new created capv cluster, the image on the control node can actually be deleted. So I think that even if we support parallel image building, it should be safe to delete images that have not been used within the last hour, right?

Reply:
But I don't think there is a risk of deleting an image we actually need, even if the job takes more than 1 hour, as long as it takes less than 1 hour to build + deliver Antrea to test Nodes (basically as long as the `deliver_antrea` function always takes less than 1 hour to execute)?

> docker system prune will remove all stopped containers. One of the reasons we do not use it is when a CAPV container encounters an error and stops, we do not want it to be deleted.

These are containers that run on the Jenkins node itself (the build node)? And if they are successful, are they deleted automatically?

Reply:
> Building the image on the capv control node typically takes around 10 minutes after the job starts. Once the image is copied to the new created capv cluster, the image on the control node can actually be deleted. So I think that even if we support parallel image building, it should be safe to delete images that have not been used within the last hour, right?

I didn't see before typing my reply. Yes this is also what I think.

Reply:
> These are containers that run on the Jenkins node itself (the build node)? And if they are successful, are they deleted automatically?

The CAPV container I mentioned refers to the container running the CAPV cluster management tools. For containers created during the image building process, yes, they will be automatically deleted after a successful build.

Reply:
> > Building the image on the capv control node typically takes around 10 minutes after the job starts. Once the image is copied to the new created capv cluster, the image on the control node can actually be deleted. So I think that even if we support parallel image building, it should be safe to delete images that have not been used within the last hour, right?
> 
> I didn't see before typing my reply. Yes this is also what I think.

Got it, PR updated.